packages/db/tests/translate/test-collections.js

```diff
@@ -1,6 +1,38 @@
 const test = require('ava');
 const { removeNilProperties } = require('@cumulus/common/util');
-const { translateApiCollectionToPostgresCollection } = require('../../dist/translate/collections');
+const {
+  translateApiCollectionToPostgresCollection,
+  translatePostgresCollectionToApiCollection,
+} = require('../../dist/translate/collections');
+
+const {
+  fakeCollectionRecordFactory,
+} = require('../../dist');
+test('translatePostgresCollectionToApiCollection converts Postgres collection to API', (t) => {
+  const meta = '{"tag_key":"tag_value"}';
+  const tags = '["tag1","tag2"]';
+  const collectionRecord = fakeCollectionRecordFactory({
+    meta,
+    tags,
+    updated_at: new Date(),
+    created_at: new Date(),
+  });
+
+  const expected = {
+    createdAt: collectionRecord.created_at.getTime(),
+    updatedAt: collectionRecord.updated_at.getTime(),
+    files: collectionRecord.files,
+    granuleId: collectionRecord.granule_id_validation_regex,
+    granuleIdExtraction: collectionRecord.granule_id_validation_regex,
+    meta: collectionRecord.meta,
```

Comment:
I commented on the translation logic as well, but while I can see how this passes testing, I'm confused about the meta/tags fields and what the expected translation there should be

Reply:
Indeed, a really good catch: 05183a1fb

Reply:
Er, actually: be59457bb

I'll explain above.